CHANGELOG.md

```diff
@@ -1,3 +1,85 @@
+
+Other enhancements
+* feat: [**Webhook for validating Configurations**](https://github.com/deislabs/akri/blob/v0.6.5/webhooks/validating/configuration/README.md) (https://github.com/deislabs/akri/pull/206)
+* feat: Support for Akri monitoring via Prometheus (https://github.com/deislabs/akri/pull/190)
+
+Misc 
+* feat: **Akri Logo** (https://github.com/deislabs/akri/pull/149)
+* fix: Allow overwriting Controller's `nodeSelectors` (https://github.com/deislabs/akri/pull/194)
+* fix: Updated `mockall` version (https://github.com/deislabs/akri/pull/214)
+* fix: Changed default image `PullPolicy` from `Always` to Kubernetes default (`IfNotPresent`) (https://github.com/deislabs/akri/pull/207)
+* fix: Improved video streaming application (for udev demo) that polls for new service creation (https://github.com/deislabs/akri/pull/173)
+* fix: Patched anomaly detection application (for OPC UA demo) to show values from all brokers (https://github.com/deislabs/akri/pull/229)
+* feat: Timestamped labels for local container builds (https://github.com/deislabs/akri/pull/234)
+* fix: Removed udev directory mount from Agent DaemonSet (https://github.com/deislabs/akri/pull/304)
+* fix: Modified Debug Echo Discovery Handler to specify `Device.properties` and added check to e2e tests (https://github.com/deislabs/akri/commit/53903e711cf4eebb9ef4367afd0f2724318b8e69#diff-97a08f11d52bb61c11fc6e159245145df32347dd439b9c6e6d94a863f3b1df4b)
+* feat: Support for specifying environment variables broker Pods via a Configuration's `brokerProperties`.
+* fix: Default memory and CPU resource requests and limits for Akri containers (INSERT PR LINK) 
+
+View the [full change log](https://github.com/deislabs/akri/compare/v0.1.5...v0.6.5)
+
+## Breaking Changes
+Akri's Configuration and Instance CRDs were modified. The old version of the CRDs should be deleted with `kubectl delete instances.akri.sh configurations.akri.sh`, and the new ones will be applied with a new Akri Helm installation.
+* Akri's Configuration CRD's `protocol` field was replaced with `discoveryHandler` in order to fit Akri's new Discovery Handler extensibility model and make the Configuration no longer strongly tied to Discovery Handlers. It's unused `units` field was removed and `properties` was renamed `brokerProperties` to be more descriptive. 
+* Akri's Instance CRD's unused `rbac` field was removed and `metadate` was renamed `brokerProperties` to be more descriptive and aligned with the Configuration CRD.
+
+Significant changes were made to Akri's Helm chart. Consult the latest user guide and Configurations documentation.
+
+By default, the Agent contains no Discovery Handlers. To deploy Discovery Handlers, they must be explicitly enabled in Akri's Helm chart.
+
+## Known Issues
+Akri's Test K3s, Kubernetes, and MicroK8s workflow periodically fails due to GitHub Runners running out of compute resources and kubelet evicting Pods. 
```

Comment:
Does this count as a known issue if it is just a part of our tests?

Reply:
i would say no ... but it seems fine to mention it

Reply:
removed due to it being on the devops side. Created an issue for it: https://github.com/deislabs/akri/issues/322